crm/modules/payment/payment.inc.php

```diff
@@ -219,7 +219,7 @@ function payment_format_currency ($value, $symbol = true) {
                 $pence = sprintf('%02d', $count);
             }
             if ($symbol) {
-                $result .= '�';
+                $result .= '£';
```

Comment:
What changed here?  It's not obvious what changed.

Reply:
the old symbolw asin't encoded porperly, so wouldn't render